acrobat/scripts/utils.js

```diff
@@ -35,3 +35,7 @@ export const [setLibs, getLibs] = (() => {
     }, () => libs,
   ];
 })();
+
+const miloLibs = setLibs('/libs');
+const { loadStyle } = await import(`${miloLibs}/utils/utils.js`);
```

Comment:
won't this slow down the initial load?

Reply:
@peller The DC scripts.js is not loading this utils.js but is already loading the milolibs utils.js so this second call would come from disk cache (if the disable cache is not explicitly disabled in deveoper tools)

Reply:
is this coupling necessary?